### PR TITLE
Added new interfaces from 1.x

### PR DESCRIPTION
**Pull request by pimjansen**

### CODE REVIEW

File: src/Extension/UserAgentExtension.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     * @example 'Mozilla/5.0 (Macintosh; U; PPC Mac OS X 10_7_1 rv:3.0; en-US) AppleWebKit/534.11.3 (KHTML, like Gecko) Version/4.0 Safari/534.11.3'
+     */
+    public function safari(): string;
+
+    /**
+     * @example 'Opera/8.25 (Windows NT 5.1; en-US) Presto/2.9.188 Version/10.00'
+     */
+    public function opera(): string;
+
+    public function windowsPlatformToken(): string;
```
Comment (bram-pkg) [inline src/Extension/UserAgentExtension.php:43]: Can we add some examples to the token functions?

Comment (pimjansen) [reply src/Extension/UserAgentExtension.php:43]: Done, they also missed in 1x so :)

File: src/Extension/UserAgentExtension.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    public function safari(): string;
+
+    /**
+     * @example 'Opera/8.25 (Windows NT 5.1; en-US) Presto/2.9.188 Version/10.00'
+     */
+    public function opera(): string;
+
+    /**
+     * @example Windows NT 6.0
```
Comment (bram-pkg) [inline src/Extension/UserAgentExtension.php]:
```suggestion
     * @example 'Windows NT 6.0'
```

File: src/Extension/UserAgentExtension.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    public function windowsPlatformToken(): string;
+
+    /**
+     * @example Macintosh; PPC Mac OS X 10_6_0
```
Comment (bram-pkg) [inline src/Extension/UserAgentExtension.php]:
```suggestion
     * @example 'Macintosh; PPC Mac OS X 10_6_0'
```

File: src/Extension/UserAgentExtension.php [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    public function macPlatformToken(): string;
+
+    /**
+     * @example iPhone; CPU iPhone OS 13_0 like Mac OS X
```
Comment (bram-pkg) [inline src/Extension/UserAgentExtension.php]:
```suggestion
     * @example 'iPhone; CPU iPhone OS 13_0 like Mac OS X'
```

File: src/Extension/UserAgentExtension.php [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    public function iosMobileToken(): string;
+
+    /**
+     * @example X11; Linux i686
```
Comment (bram-pkg) [inline src/Extension/UserAgentExtension.php]:
```suggestion
     * @example 'X11; Linux i686'
```

**Review by bram-pkg — APPROVED** [review]

**Pull request merged**